server/libs/infra/src/db/config/database.config.ts

```diff
@@ -11,11 +11,11 @@ const baseDatabaseConfig: PostgresConnectionOptions = {
 };
 
 const envBasedDatabaseConfig = {
-  host: process.env.DB_HOSTNAME || 'immich_postgres',
```

Comment:
Is this related to development environment changes?

Reply:
With the default config like this, we can run typeorm commands from the host rather than within the server container, it couples with the nicer typeorm commands in package.json now. We always tell the user to set them in the .env file, so I think changing the default here is fine.